app/2.0/docs/tools/polymer-cli.md

```diff
@@ -39,500 +39,28 @@ You're all set. Run `polymer help` to view a list of commands.
 
 ## Overview {#overview}
```

Comment:
And maybe `## Install` => `## Install Polymer CLI`?

Not sure. Do what feels right.